fiftyone/server/routes/get_similar_labels_frames.py

```diff
@@ -272,7 +272,7 @@ async def post(self, request: Request, data: dict):
         view = fov.make_optimized_select_view(view, sample_id, flatten=True)
 
         filtered_schema = view.get_frame_field_schema(
-            flat=True, embedded_doc_type=fol.Instance
+            flat=True, embedded_doc_type=fol.Label
```

Comment:
Do you specifically need to check for `Detection(s)|Polyline(s)|Keypoint(s)` fields here? 

Or are we specifically now being generic enough to handle `Classification|Segmentation|Heatmap` fields who get `instance` attributes added to them? 

btw I haven't inspected

Reply:
Addressed in [3b5fe09](https://github.com/voxel51/fiftyone/pull/5815/commits/3b5fe090cf1f59cb732eb07edb67b09cac637d6e)